tests/unit/Mvc/Router/AnnotationsTest.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace Phalcon\Test\Unit\Mvc\Router;
+
+use Phalcon\Test\Module\UnitTest;
+
+require_once TESTS_PATH . '/../unit-tests/controllers/AnnotationController.php';
```

Comment:
This file should be moved to te `tests/_data/controllers`. Also take a look at `tests/_config/bootstrap.php` (`$loader->registerNamespaces`)